src/test/java/com/microsoft/sqlserver/testframework/DBResultSet.java

```diff
@@ -213,7 +210,7 @@ public void verifydata(int ordinal,
 
         // Verify
         // TODO: Check the intermittent verification error
-        // verifydata(ordinal, coercion, expectedData, retrieved);
+        verifydata(ordinal, coercion, expectedData, retrieved);
```

Comment:
does the comment still need to be there?

Reply:
fixed